Dockerfile

```diff
@@ -2,27 +2,30 @@
 # BUILDER #
 ###########
 FROM clinicalgenomics/python3.12-venv:1.0 AS python-builder
```

Comment:
Do we still need this base image with the virtualenv now? Because I don't think so?